src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -227,7 +227,7 @@ internal IEnumerable<string> CreateArgument()
             {
                 foreach(var arg in this.VSTestCollect)
                 {
-                    allArgs.Add("--collect:" + arg);
+                    allArgs.Add("--collect:" + arg.AddDoubleQuote());
```

Comment:
Add quotes for `VSTestLogger` parameter too, check is there any other parameter need quotes?